client/package.json

```diff
@@ -112,8 +112,7 @@
     "react-overlays": "^5.1.1",
     "react-player": "^2.10.0",
     "react-redux": "^5.1.2",
-    "react-router": "^4.3.1",
-    "react-router-dom": "^4.3.1",
+    "react-router-dom": "^6.3.0",
```

Comment:
Not this line but can check if the `history` dependency above is still needed?

Reply:
We can keep it as of now. react-router depends on history anyways